freebsd/Platform.c

```diff
@@ -313,6 +313,7 @@ bool Platform_getDiskIO(DiskIOData* data) {
    int count = current.dinfo->numdevs;
 
    unsigned long long int bytesReadSum = 0, bytesWriteSum = 0, timeSpendSum = 0;
+   unsigned long long int numDisks = 0;
 
```

Comment:
Possible cleanup: Be consistent with later usage of this variable and use `uint64_t` here too …